### DISCUSSION TITLE
Error calling os. system() command neper using Python

### DISCUSSION THREAD

**Discussion by zhfzzz** [neperfepx/neper#767] [Q&A] [answered]
Error calling os. system() command neper using Python,
example：
os.system(f'neper -T -n 1000 -id 1 -domain "cube(1.0,1.0,0.025)" -o data/domain -format tess,obj,ori')
return error
May I know how to call this command，thanks.
When searching for answers online, some people believe that the reason is due to spaces, but without spaces, the neper command cannot be used.
Dear, I need your help.

**Comment by rquey** [comment]
If the `data` directory does not exist, it is probably it.

Otherwise, the Neper command itself looks okay, so it must be a Python issue and hopefully there is someone out there to help...

**Comment by mkasemer** [comment, marked as answer]
@kshankar7 or @emengiste - any thoughts on the above Python command?

My use of the `os` package in Python is limited, but a quick google search shows examples where the command has spaces:
```
import os

cmd = "git --version"

returned_value = os.system(cmd)  # returns the exit code in unix
print('returned value:', returned_value)
```

**EDIT:** Is it the (seemingly) erroneous `f` at the beginning of your command? i.e., `os.system(f'...')` ?
